core/tests/performance_tests/test_config.py

```diff
@@ -32,17 +32,131 @@
 """
 
 SPLASH_PAGE_KEY = 'splash'
+LIBRARY_PAGE_KEY = 'library'
+CREATOR_DASHBOARD_KEY = 'dashboard'
+EXPLORATION_PLAYER_KEY = 'explore'
+EXPLORATION_EDITOR_KEY = 'editor'
+COLLECTION_PLAYER_KEY = 'collection'
+PROFILE_PAGE_KEY = 'profile'
 
 TEST_DATA = {
```

Comment:
I don't see anything in here with a preload_option of 'username', yet this preload option is referenced elsewhere. Which is correct?